app/assets/javascripts/graphs.coffee

```diff
@@ -0,0 +1,155 @@
+$ ->
+  if gon.raw_run isnt `undefined`
```

Comment:
ah, i think that's from when i ported everything from javascript to coffeescript and used a converter to do it. feel free to fix that too, or if not i'll get it later